code/game/turfs/openspace/openspace.dm

```diff
@@ -0,0 +1,62 @@
+/turf/open/openspace

```

Comment:
Is there some reason this isn't inheriting from `/turf/open/space`?

Reply:
plenty of use cases (originally made for planetstation project) that don't involve space tiles but in reality I don't know how much of the space code can be reused for this and it was easier to just make something brand new.
however making it /turf/open/space/openspace would make it much easier to implement multiz stations I guess.